static/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -431,6 +432,14 @@ export class IntegrationListDirectory extends AsyncComponent<
         publishStatus={app.status}
         configurations={0}
         categories={categories}
+        customIcon={
+          <PluginIcon
```

Comment:
@leeandher It might make sense to build a new component called `SentryAppIcon` that can handle the fallback logic that's only used for `SentryApps`. It can render a `PluginIcon` or an `Avatar` depending on the feature flag and whether that `SentryApp` has the avatar available.

Reply:
Built out! `SentryAppIcon` was the name of the component we used for the issue details page (B&W logos in the sidebar, etc). I renamed that to a more accurate `SentryAppComponentIcon` and created `SentryAppIcon` as per your suggestion.